### ISSUE TITLE
WF spec mentions invalid nodeStatus() EL function

### ISSUE THREAD

**Issue by tucu00** [YahooArchive/oozie#214]
The "3.1.4 Decision Control Node" uses it as an example

**Comment by tucu00** [comment]
More inaccuracies in the WF spec

**Comment by wagz211** [comment]
the bad sections are:

```
${wf:nodeStatus('myfirstjob') == 'OK'}

${fs:size('/usr/foo/myinputdir') gt 10 * GB}
```

and:

Example:

<workflow-app name="foo-wf" xmlns="uri:oozie:workflow:0.1">
    ...
    <decision name="mydecision">
        <switch>
            <case to="reconsolidatejob">
              ${fs:size(secondjobOutputDir) gt 10 \* GB}
            </case>
            <case to="rexpandjob">
              ${fs:size(secondjobOutputDir) lt 100 \* MB}
            </case>
            <case to="recomputejob">
              ${ hadoop:counters('secondjob')[RECORDS][REDUCE_OUT] lt 1000000 }
            </case>
            <default to="end"/>
        </switch>
    </decision>
    ...
</workflow-app>

**Comment by tucu00** [comment]
Closed by cdeaf4c3148ac069540f35678771671319a4dcec WF spec mentions invalid nodeStatus() EL function
